### DISCUSSION TITLE
Flat File VAT Invoice Data Report (VIDR) report always returns empty result

### DISCUSSION THREAD

**Discussion by maniicandy** [amzn/selling-partner-api-models#2808] [General]
HI,

In the Selling partner API, This report type "GET_FLAT_FILE_VAT_INVOICE_DATA_REPORT" always returns a empty result
does anyone use this report type and got a result ?

**Comment by michelexport** [comment]
Hi there, yes we do and we get results.
Do you have the correct developer role to access this report?
